scylla-cql/src/frame/response/result.rs

```diff
@@ -836,6 +836,13 @@ impl DeserializedMetadataAndRawRows {
         );
         TypedRowIterator::new(raw)
     }
+
+    /// Allows to retrieve raw rows, without the need for deserialization
+    /// Intended to be used only in nodejs-rs driver only.
+    #[cfg(nodejs_rs_unstable)]
+    pub fn get_raw_rows(&self) -> &Bytes {
+        &self.raw_rows
+    }
```

Comment:
:wrench: https://rust-lang.github.io/api-guidelines/naming.html#c-getter

Reply:
Actually, I think this function could be added without any `cfg` attached. It won't be accessible to driver's users anyway, as the very `DeserializedMetadataAndRawRows` is not accessible by them.
@Lorak-mmk WDYT?